public-docs/installation-reaction-platform.md

```diff
@@ -0,0 +1,105 @@
+---
+id: installation-reaction-platform
+title: Install with Reaction Platform
+---
+
+Follow these instructions to run the entire Reaction Platform. 
+
+The Platform will install and run the the entire suite of Reaction services in the these directories:
+
+| Directory                                                                                  | Services                        |
+| ------------------------------------------------------------------------------------------ | ------------------------------- |
+| [`reaction`](https://github.com/reactioncommerce/reaction)                                 | GraphQL API, Classic UI, Mongo  |
+| [`reaction-hydra`](https://github.com/reactioncommerce/reaction-hydra)                     | Authentication server, Postgres |
+| [`reaction-next-starterkit`](https://github.com/reactioncommerce/reaction-next-starterkit) | Next.js storefront application  |
+
+## Installation
+
+1. Requirements:
+
+- Install Docker and Docker Compose ([Mac](https://docs.docker.com/docker-for-mac/install/) | [Windows](https://docs.docker.com/docker-for-windows/install/)<sup>*</sup> | [Linux](https://docs.docker.com/compose/install/#install-compose)<sup>**</sup>) 
+- Install [Node.js](https://nodejs.org/en/)
+- A GitHub account with a configured SSH key
+
+> <sup>*</sup>Windows: Reaction Platform has not been fully tested in Windows at this time.
+
+> <sup>**</sup>Linux: Docker Compose is included when installing Docker on Mac and Windows, but will need to be installed separately on Linux.
+
+1. Before you get started, make sure you are not running any applications the necessary ports: `3000`, `4000`, `4444`.
+
+2. Clone [**Reaction Platform**](https://github.com/reactioncommerce/reaction-platform)
+
+```sh
+git clone git@github.com:reactioncommerce/reaction-platform.git
+cd reaction-platform
+```
+
+4. Now you're ready to install. Run this command to bootstrap and start all of the services:
+
+```sh
+make
+```
+
+This process may take some time. The Platform is checking that dependencies are present, cloning the sub-projects from GitHub, downloading and building Docker images, and starting services.
+
+5. As the various repositories are downloading and services are starting, check the logs by running `docker-compose logs -f` in the specific repository. For example, to display the `reaction` logs, run:
+
+```sh
+cd reaction
+docker-compose logs -f
+```
+
+To display the `reaction-next-starterkit` logs, run:
+
+```sh
+cd reaction-next-starterkit
+docker-compose logs -f
+```
+
+6. Once the initial `make`  process finishes, the services will be accessible at the following URLs:
```

Comment:
Probably need to add a note here about expected wait time. Immediately after `make` finishes, there's still a long wait before the web apps are running.